Examples/Detectors/ContextualDetector/src/ExternalAlignmentDecorator.cpp

```diff
@@ -1,14 +1,9 @@
-// This file is part of the Acts project.
-//
-// Copyright (C) 2019 CERN for the benefit of the Acts project
-//
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
 #include "ActsExamples/ContextualDetector/ExternalAlignmentDecorator.hpp"
+<<<<<<< HEAD
 
 #include "Acts/Geometry/GeometryContext.hpp"
+=======
+>>>>>>> 2c318123d (adding hpp and cpp files-xternal alignment)
```

Comment:
These are git "conflict markers", and should be removed otherwise the code will not compile. C.f. https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging#_basic_merge_conflicts